src/opencl_sha3.cl

```diff
@@ -52,16 +52,12 @@ kernel void sha3(global ulong *buffer, ulong nonce_start, ulong difficulty,
     uint r, x, y, t;
     ulong tmp, current, C[5];
     for (r = 0; r < 24; ++r) {
-      for (x = 0; x < 5; ++x) {
-        C[x] = state[x] ^ state[x + 5] ^ state[x + 10] ^ state[x + 15] ^
-               state[x + 20];
-      }
-      for (x = 0; x < 5; ++x) {
-        tmp = C[(x + 4) % 5] ^ rotate(C[(x + 1) % 5], 1ul);
-        for (y = 0; y < 5; ++y) {
-          state[x + y * 5] ^= tmp;
```

Comment:
In the previous code, the newly computed values were updated into `state`. The new unrolled code never does this, and the result is finding blocks with a difficulty far too low to submit.